### PR TITLE
Sprint a

### PR DESCRIPTION
**Pull request by ptrthegr8**

### CODE REVIEW

File: submissions/sprint_a/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def configure_app():
+    import os
+    import dotenv
+    from flask import Flask
+    PROJECT_ROOT = os.path.dirname('.')
```
Comment (infosmith) [inline submissions/sprint_a/__init__.py]: use os.path.dirname(os.path.abspath(\_\_file__)), is safer

File: submissions/sprint_a/app.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from sprint_a import app
+from flask import jsonify
```
Comment (infosmith) [inline submissions/sprint_a/app.py:2]: Check the order of your imports: https://www.python.org/dev/peps/pep-0008/#imports

Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.

**Review by infosmith — CHANGES_REQUESTED** [review]
Please add docstrings and a shebang to your files.

**Comment by itsthejoker** [comment]
:+1:

**Pull request closed without merging**